### PR TITLE
Worker landing page

### PR DESCRIPTION
**Pull request by alexgtang**
## Description of changes
Redesigned the worker landing page. Added cards and pagination. Can click on cards to view client profile.
## Screenshot(s) or GIF(s) of changes
image

### CODE REVIEW

**Comment by alexgtang** [comment]
https://github.com/keepid/keepid_client/issues/323

File: src/static/styles/App.scss [hunk 1]
```diff
@@ -702,3 +702,108 @@ div.scrollbar {
...
+
+.right-angle-left {
+	border-bottom-left-radius: 0rem;
+	border-top-left-radius: 0rem;
+}
+
+.pagination-link {
+	color: $primary-theme;
+	float: left;
+	padding: 8px 16px;
```
Comment (crchong1) [inline src/static/styles/App.scss:744]: small nits but still try to convert px to rem

File: src/static/styles/App.scss [hunk 2]
```diff
@@ -702,3 +702,108 @@ div.scrollbar {
...
+	border-left: none;
+}
+
+.pagination-link-1 {
+	color: $primary-theme;
+	float: left;
+	padding: 8px 16px;
+	text-decoration: none;
+	transition: background-color .3s;
+	border: 1px solid #ECEEEF;
```
Comment (crchong1) [inline src/static/styles/App.scss:757]: we should reuse color variables as much as possible- can you make a grey variable if it doesn't already exist

File: src/components/LandingPages/WorkerLanding.tsx [hunk 3]
```diff
@@ -183,85 +207,108 @@ class WorkerLanding extends Component<Props, State> {
...
-                  type="button"
-                  className="btn btn-dark mb-2 btn-sm"
-                  onClick={(event) =>
-                    this.handleClickSendApplication(event, client)
-                  }
-                >
-                  Send Application
-                </button>
+        <div key={client.username} className="card client-card mb-4 mr-4 flex-column">
+          <div className="dropdown" style={{ position: 'absolute', top: 24, right: 24 }}>
```
Comment (crchong1) [inline src/components/LandingPages/WorkerLanding.tsx:228]: if possible, lets not have in-line styling inside the html. If needed, we can define new CSS classes and have them apply to a bunch of areas at once.

**Review by crchong1 — CHANGES_REQUESTED** [review]
just left a couple of small css comments

**Comment by crchong1** [comment]
also can you look into why you tests are failing on circleci? that might be a bit alarming

**Comment by crchong1** [comment]
sorry one last comment- can you revert the behavior of the search bar so it will search on type?

**Review by crchong1 — APPROVED** [review]
Just left one last comment, otherwise lgtm

**Comment by alexgtang** [comment]
Yep! The search bar now searches on type. I also gave the workers the option to just click 'Search' which will display all results.

**Pull request merged**